lib/databaseconnection.rb

```diff
@@ -0,0 +1,12 @@
+require 'pg'
```

Comment:
This is really useful way of ensuring you don't continuously have to keep typing the db connection